docs/docs-beta/docs/guides/external-systems/pipes.md

```diff
@@ -1,5 +1,33 @@
 ---
-title: "Excuting code outside of Dagster with Pipes"
+title: "Executing code outside of Dagster with Pipes"
 sidebar_position: 40
 sidebar_label: "External execution with Pipes"
----
\ No newline at end of file
+---
+
+# Executing code outside of Dagster with Pipes
+
+Dagster Pipes provides a powerful mechanism for invoking code outside of Dagster, while providing all the benefits of scheduling, reporting, and observability of native Dagster pipelines.
+
+In this guide, we'll walk you through how to invoke non-Dagster code through Pipes.
+
+<details>
+<summary>Prerequisites</summary>
+
+- Familiarity with [Assets](/concepts/assets)
+</details>
+
+## Setting up an asset that invokes your external code
+
```

Comment:
potentially, we can also need to explain the file structure here? as in, the script can live anywhere

Reply:
changed the external code location to be `/usr/bin/external_code.py` as a cue